src/lib/Scenes/Inbox/Components/Conversations/MakeOfferModal.tsx

```diff
@@ -30,6 +31,12 @@ export const MakeOfferModal: React.FC<MakeOfferModalProps> = ({ ...props }) => {
     }
   }
 
+  useEffect(() => {
+    if (editionSets?.length === 1) {
+      setSelectedEdition(editionSets[0]?.internalID)
+    }
+  })
```

Comment:
```suggestion
  }, editionSets)
```
Wondering if it is possible to just run the effect only the `editionSets` change.

Reply:
should be `[editionSets]`, no? Or `[]` if we only want it to run once on mount- probably no difference in this case and one is more explicit.

Reply:
Yes! Good catch, @erikdstock. I think, in this case, either solution would work.